tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -31,23 +31,23 @@ Debug Options -> Enable Debug Output Features to y
 Debug Options -> Enable backtracking using Frame pointer register  to y
 ```
 
-## How to upload RAMDUMP
+## How to upload RAMDUMP/FSDUMP
 ### In Linux
-With the RAMDUMP configured above, whenever the target board crashes because an assert condition, it enters PANIC mode, and displays the following message:  
+With the DUMP Tool configured above, whenever the target board crashes because an assert condition, it enters PANIC mode, and displays the following message:  
```

Comment:
I think `whenever the target board crashes because an assert condition` should be changed to `whenever the target board crashes because of an assert condition`

Reply:
Agreed, updated.